api/src/main/java/org/pmiops/workbench/api/WorkspacesController.java

```diff
@@ -580,20 +579,20 @@ public ResponseEntity<CloneWorkspaceResponse> cloneWorkspace(
     // Note: It is possible for a workspace to be (partially) created and return
     // a 500 to the user if this block of code fails since the workspace is already
     // committed to the database in an earlier call
-    Map<String, WorkspaceAccessLevel> clonedRoles = new HashMap<>();
     if (Boolean.TRUE.equals(body.getIncludeUserRoles())) {
-      Map<String, RawlsWorkspaceAccessEntry> fromAclsMap =
-          workspaceAuthService.getFirecloudWorkspaceAcls(
+      var fromAcl =
```

Comment:
This looks more readable, to me.  Please confirm that the logic is the same.

I'll be coming back to this in a later PR to make a change.

Reply:
It looks the same logic, it is cleaner.  Not saying its incorrect, but i have to get used to var, rather than explicitly declared variable type

Reply:
OK to ask for non-`var` when appropriate